src/ORM/DataObject.php

```diff
@@ -2228,15 +2236,20 @@ public function getManyManyComponents($componentName, $id = null)
     }
 
     /**
-     * Return the class of a one-to-one component.  If $component is null, return all of the one-to-one components and
-     * their classes. If the selected has_one is a polymorphic field then 'DataObject' will be returned for the type.
+     * Return the class of a all has_one relations.
      *
-     * @return string|array The class of the one-to-one component, or an array of all one-to-one components and
-     *                          their classes.
+     * @return array An array of all has_one components and their classes.
```

Comment:
This was just a plain lie before. Updated to indicate the actual functionality of this method.